Added dutch translations

KingToadz opened the pull request:
Hi,

I added a dutch translation based on the english word set with Google translate.
- Removed words with spaces
- Sorted words alphabetically
- Removed duplicates

I removed words with spaces because the translation would make some bad combinations.

piteur commented:
Nice! 🎉 

I recently added the French ones and removed the words tagged with `#i` which means `impossible`.
If we want to keep the translations ISO between each other, feel free to take a look at the feedback I got there: https://github.com/scribble-rs/scribble.rs/pull/98

KingToadz commented:
I updated the wordlist by removing the impossible words and I have removed the difficulty from other words. This als fixed a the wordlist with duplicate words with different difficulty. As far as I checked the wordlist didn't contain any plural words 😄

Bios-Marcel commented:
Cool, thanks. Good job :)
